clientSide/womeneducation/src/Components/Footer/Footer.css

```diff
@@ -0,0 +1,45 @@
+.line
+{
+    margin-left: 100px;
+}
+.heartsymbolsize
+{
+    width:15px;
+    height:13px;
+    margin-left: 5px;
+    margin-right: 5px;
+}
+.footertext
+{
+    font-weight: bold;
+    text-align: center;
+}
+.line
+{
+    font-size:17px;
+}
+.heartsymbolsize
+{
+    width:15px;
+    height:13px;
+    margin-left: 5px;
+    margin-right: 5px;
+}
+.footertext
+{
+    font-weight: bold;
+    text-align: center;
```

Comment:
same line 13-15 is also same please use same class names

Reply:
I removed same classname